script/test

```diff
@@ -219,19 +219,28 @@ do_expect()
         test_patterns=(-name 'cli-*.exp' -o -name 'simulation-*.exp')
     fi
 
+    local temp_dir
+    temp_dir=$(mktemp -d)
```

Comment:
create a temp file instead.

Reply:
Done. Now using current directory to save the log file.